README.md

```diff
@@ -2289,9 +2289,9 @@ Pass `-help` to VictoriaMetrics in order to see the list of supported command-li
   -http.shutdownDelay duration
      Optional delay before http server shutdown. During this delay, the server returns non-OK responses from /health page, so load balancers can route new requests to other servers
   -httpAuth.password string
-     Password for HTTP Basic Auth. The authentication is disabled if -httpAuth.username is empty
+     Password for HTTP servers' Basic Auth. The authentication is disabled if -httpAuth.username is empty
```

Comment:
Is it a typo `HTTP servers' Basic Auth` -> `HTTP server's Basic Auth`?

Reply:
Yes, thanks for noticing, updated